elections/2025/governance-committee-election.md

```diff
@@ -0,0 +1,105 @@
+
+The election committee will not merge the Pull Request until the candidate has confirmed their desire to be nominated (if not self-nominating) and has been ratified via PR comments.
+
+Nominees should also send their email address to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io) – it will be kept private and used only for candidate communications as the election process proceeds.
+
+The Governance Committee or appointed people will contact every nominee directly to ensure the commitment and desire to be nominated.
+
+# Voter Eligibility
+
+All [members of standing](../../governance-charter.md#members-of-standing) will automatically be eligible to vote. To confirm your eligibility status, see the [election announcements issue](https://github.com/open-telemetry/community/issues/3001). If your code contributions do not meet eligibility requirements, but you believe your non-code contributions should make you eligible to vote, you can request an exemption by submitting an exemption [request form](https://docs.google.com/forms/d/e/1FAIpQLSeSA09xDIv0uyb6vrP8xBbLjm8NsgihrG8GHxacbigF17sNDw/viewform?usp=dialog).
+
+One of two options will be available in the form to prove eligibility:
+
+* GitHub handle
+* List either:
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the charter](../../governance-charter.md#members-of-standing): particularly, your relationship to a well-known project that's taken a hard dependency on OpenTelemetry, or your involvement with a well-known organization's adoption of OpenTelemetry as an end-user.
+
+All exemption forms are private. Only the current governance and election committee members will have access to this information. The governance committee will discard the lists one month after the election.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/elections/f94a7c58-990b-11f0-a16d-5270fb641b4c) using their GitHub account. Voting will be approval voting, where each voter may select up to four candidates. The four candidates with the most votes win the election.
+
+The election committee will accept late registrations to vote and requests to re-send the voting link via email to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io). We encourage pre-registration to minimize the effort required to run this election.
+
+Per Helios Voting, voting is entirely private: nobody will know any individual's vote.
+
+# Results
+
+Voting will close on 29 October 2025 at 23:59 UTC. Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
```

Comment:
Have we considered closing voting on "29 October 2025, End of Day, [Anywhere On Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth) (30 October 2025 at UTC 11:59)"?

Not sure if it matters much given that we have multiple days but it's a bit easier to understand maybe (you have to vote before EoD October 29th no matter where you live)

Reply:
I like this idea! I will update Helios and this text but will leave this comment open. Reviewers, please let me know if you object!

Reply:
```suggestion
Voting will close at the end of the day on 29 October 2025 (technically, 23:59 in the International Date Line West time zone / 30 October 2025 noon UTC). Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
```